src/test/java/ladder/LadderGameResultTests.java

```diff
@@ -0,0 +1,89 @@
+package ladder;
+
+import ladder.model.*;
+import ladder.model.dto.LadderGameMemberResult;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+
+@DisplayName("사다리 게임 실행 결과 테스트")
+public class LadderGameResultTests {
+
+    private Members members;
+    private Ladder ladder;
+    private LadderGameRewords ladderGameRewords;
+    private LadderGameRewords expectedLadderGameRewordsResult;
+    private LadderGameExecutionInfo ladderGameExecutionInfo;
+
+    @BeforeEach
+    public void init() {
+        List<LadderLine> ladderLines = new ArrayList<>();
+        ladderLines.add(LadderLine.newInstance(LadderBridge.EXIST, LadderBridge.UN_EXIST, LadderBridge.EXIST));
+        ladderLines.add(LadderLine.newInstance(LadderBridge.UN_EXIST, LadderBridge.EXIST, LadderBridge.UN_EXIST));
+        ladderLines.add(LadderLine.newInstance(LadderBridge.EXIST, LadderBridge.UN_EXIST, LadderBridge.UN_EXIST));
+        ladderLines.add(LadderLine.newInstance(LadderBridge.UN_EXIST, LadderBridge.EXIST, LadderBridge.UN_EXIST));
+        ladderLines.add(LadderLine.newInstance(LadderBridge.UN_EXIST, LadderBridge.EXIST, LadderBridge.UN_EXIST));
```

Comment:
```suggestion
        ladderLines.add(newInstance(EXIST, UN_EXIST, EXIST));
        ladderLines.add(newInstance(UN_EXIST, EXIST, UN_EXIST));
        ladderLines.add(newInstance(EXIST, UN_EXIST, UN_EXIST));
        ladderLines.add(newInstance(UN_EXIST, EXIST, UN_EXIST));
        ladderLines.add(newInstance(UN_EXIST, EXIST, UN_EXIST));
```

테스트를 위해 초기 데이터를 세팅하는 부분이기도 하고, static import를 사용해도 의미를 분명히 전달할 수 있을거 같아요